snaps_openstack/ansible_p/commission/openstack/playbooks/deploy_mode/kolla/kolla_set_pin.yaml

```diff
@@ -119,5 +135,14 @@
       args:
        executable: /bin/bash
       async: 900
+      poll: 5
+      register: out
+      delegate_to: "{{CONTROLLER_IP}}"
       ignore_errors: True
+    - debug: var=out.stdout_lines
+    - debug: var=out.stderr_lines
+
+    - name: Validating task [run init-runonce] returned 0
+      fail: msg="run init-runonce command failed"
+      when: out.rc != 0
```

Comment:
Please fix how to validate success of an async task (see commit https://github.com/cablelabs/snaps-openstack/commit/e2c7d6654f54663a009286634c3be15555d681c6)